presentation/matrix_view.py

```diff
@@ -261,17 +253,15 @@ def make_cell_hover_text(self, from_agent_id, to_agent_id: AgentID, mesh: MeshRe
         distance_unit = self._config.distance_unit
         distance = calc_distance(from_agent.coords, to_agent.coords, distance_unit)
 
-        cell_hover_text = [
-            f"{from_agent.alias} -> {to_agent.alias}",
-            f"Distance: {distance:.0f} {distance_unit.value}",
-        ]
+        cell_hover_text = [f"{from_agent.alias} -> {to_agent.alias}", f"Distance: {distance:.0f} {distance_unit.value}"]
 
         if conn.has_no_data():
             cell_hover_text.append("NO DATA")
         else:
             cell_hover_text.append(f"Latency: {conn.latency_millisec.value:.2f} ms")
             cell_hover_text.append(f"Jitter: {conn.jitter_millisec.value:.2f} ms")
             cell_hover_text.append(f"Loss: {conn.packet_loss_percent.value:.1f}%")
+            cell_hover_text.append(f"Last updated: {conn.last_updated_utc.strftime('%x %X')}")
```

Comment:
We will need to figure out a way to convert the time to client's local time (which will likely require some client-side JavaScript).

In the meantime add indication of timezone to the string `strftime('%x %X %Z')`

Reply:
Adding time zone information